src/Graph.hs

```diff
@@ -382,9 +382,9 @@ callMapFromNameMap = CallMap . Map.mapWithKey fromEntry
 extractPermissionActions :: [CAttr] -> PermissionActionSet
 extractPermissionActions attributes = runIdentity . fmap HashSet.fromList . runListT $ do
   CAttr WardKeyword expressions _ <- select attributes
-  CCall (CVar (Ident actionName _ _) _) permissions pos <- select expressions
+  CCall (CVar (Ident actionName _ _) _) permissions _pos <- select expressions
   permissionSpec <- select permissions
-  (permission, mSubject) <- case permissionSpec of
+  (permission, _mSubject) <- case permissionSpec of
```

Comment:
I think I left this unmarked with `_` because it was supposed to be the “hook” where we start to handle #37, albeit with `CVar` instead of `CConst`